dev/com.ibm.ws.concurrent.mp_fat/test-applications/MPConcurrentApp/src/concurrent/mp/fat/web/MPConcurrentTestServlet.java

```diff
@@ -5311,6 +5314,7 @@ public void testWhenCompleteAfterSuccessfulCompletion() throws Exception {
      * than a managed executor.
      */
     @Test
+    @SkipForRepeat(MPContextProp11RepeatAction.ID)
```

Comment:
@njr-11 This test fails with version 1.1, when you review please decide if a follow up change is needed to fix that.

Reply:
There ought to be a difference between 1.1 and 1.0 for this test given that its description states: "Verify ... that Async operations requested without an executor parameter raise UnsupportedOperationException, as required by the spec". That is a requirement of the 1.0 specification to fail with unsupported, whereas the 1.1 specification adds new capability here such that the Async operations are now supported.  It looks like you went above and beyond in enabling some additional test classes to run with 1.1, which is fine, it just requires the skip here.